README.md

```diff
@@ -32,22 +33,231 @@ scheduled_task { 'Run Notepad':
 
 ## Usage
 
-See the [Puppet resource documentation](https://puppet.com/docs/puppet/latest/types/scheduled_task.html) for more information.
+One common use is to kick off a script, either once or on a regular cadence.
+In this first example we schedule a cleanup script to run this one time.
+
+~~~ puppet
+scheduled_task { 'Disk Cleanup': # Unique name for the scheduled task
+  command => 'C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe -File "C:\\Scripts\Clear-DiskSpace.ps1"',
+  enabled => 'true',             # This is the default, but including it is good practice. Flip to 'false' to disable the task.
+  trigger => [{
+    schedule   => 'once',        # Defines the trigger type; required.
+    start_time => '23:20',       # Defines the time the task should run; required.
+    start_date => '2018-01-01'   # Defaults to the current date; not required.
```

Comment:
Is it proper or not proper to have a trailing comma after the last item in these hashes? I haven't tried it.

Reply:
Puppet code allows trailing commas, like Ruby.